packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -374,9 +384,8 @@ export async function ensureAvailableUsers(
     //check if event time is within the date range
     for (const dateRange of dateRanges) {
       if (
-        (dayjs.utc(input.dateFrom).isAfter(dateRange.start) ||
-          dayjs.utc(input.dateFrom).isSame(dateRange.start)) &&
-        (dayjs.utc(input.dateTo).isBefore(dateRange.end) || dayjs.utc(input.dateTo).isSame(dateRange.end))
+        (startDateTimeUtc.isAfter(dateRange.start) || startDateTimeUtc.isSame(dateRange.start)) &&
+        (endDateTimeUtc.isBefore(dateRange.end) || endDateTimeUtc.isSame(dateRange.end))
```

Comment:
Found another `No available users found` error (not new, but sill exists in this branch)

When booking the slot before midnight it is throwing 'No available users found'
dateRange.end is 23:29 but endDateTimeUtc is 00:00 (so `endDateTimeUtc.isSame(dateRange.end) ` is false but should be true)